adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -219,27 +219,14 @@ def _parse_remote_config(config: Dict[str, Any], parsed_config: Dict[str, Any])
 
 
 def _get_dbt_packages(
+    adapter_type: str,
     is_teleport: bool = False,
     is_remote: bool = False
 ) -> Iterator[Tuple[str, Optional[str]]]:
-    # package_distributions will return a mapping of top-level package names to a list of distribution names (
-    # the PyPI names instead of the import names). An example distirbution info is the following, which
-    # contains both the main exporter of the top-level name (dbt-core) as well as all the packages that
-    # export anything to that namespace:
-    #   {"dbt": ["dbt-core", "dbt-postgres", "dbt-athena-adapter"]}
-    #
-    # This won't only include dbt.adapters.xxx, but anything that might export anything to the dbt namespace
-    # (e.g. a hypothetical plugin that only exports stuff to dbt.includes.xxx) which in theory would allow us
-    # to replicate the exact environment.
-    package_distributions = importlib_metadata.packages_distributions()
-    for dbt_plugin_name in package_distributions.get("dbt", []):
+    dbt_adapter = f"dbt-{adapter_type}"
```

Comment:
Does this work for athena and other non-standard adapter?

Reply:
Great question! We should expand our integration tests to include athena, etc.